README.md

```diff
@@ -1,8 +1,79 @@
-# HashiCorp Marketing Webdev Challenge!
+# HashiCorp Web Presence Engineering Challenge
+
+### Getting started
 
 Run `npm install` then `npm start`, and visit `http://localhost:3000` for instructions.
 
-## Submitting Your Work
+### Introduction
+
+Hi! Thanks so much for interviewing with us 💖. If you are reading this, it means that we like you, and I hope you like us too! In this next step, you get an opportunity to see how we work, and we get the same opportunity for you. Let's get right to it.
+
+The task is to build a "people directory" page. This page should allow visitors to search by name for a person. If this were a real page, we'd want to search all 317 records in our CMS, but for the purposes of this exercise, limit your results to 100. This is the maximum number of results that our CMS API will return in a single page.
+
+Use [this Figma file](https://www.figma.com/file/xGicP4qkXbMhte4LAYxC4X/Untitled?node-id=0%3A1) as your source of truth for design and behavior:
+
+_**Note:** If you are unable to inspect elements in the design to get exact values for colors and font styles, make sure you're signed in with a Figma account, as we will expect your submission to match the design as closely as possible._
+
+There are definitely some parts of this that are a little ambiguous, which is how real world projects always work. Treat this the same way you would if you started working with us and this was your first day — this is quite similar to the type of task we would assign a new hire on their first day. We will set up a Discord channel for you to ask questions, get clarification, and collaborate with us in general, and we strongly encourage you to take advantage of this.
+
+We expect this to be done within a day or two's worth of full-time work - about 8 - 14 hours. We know that you likely have other commitments, and so we generally look at about week for candidates to complete the challenge. If you're approaching 14 hours of total time spent, just stop where you are and we'll complete the evaluation based on what you have done.
+
+### Using the Discord channel
+
+You will have been added to a private discord channel with a few members of our team. The purpose of this channel is to give a space for us to collaborate together, again, to simulate us working together. Use the channel to ask questions, get clarity on anything ambiguous and share your approach to the work. Asking questions and being active in the channel are positive things, not negative. We don't expect you to have all the answers! There aren't any tricks in this challenge, but just like in real work, there are areas of ambiguity so if you're completing the challenge without any questions, you should reconsider.
+
```

Comment:
This is a net new addition.

Reply:
> you should reconsider.

Could we be more specific here? Maybe "reconsider your assumptions and collaborate with the team to ensure the best possible outcome" or something to that effect?

Reply:
Putting that wording in this PR, before merging.